operations/pyroscope/helm/pyroscope/templates/services.yaml

```diff
@@ -9,6 +10,10 @@ metadata:
   labels:
     {{- include "pyroscope.labels" . | nindent 4 }}
     app.kubernetes.io/component: {{ $component | quote }}
+  annotations:
+  {{- with $values.serviceAnnotations }}
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
```

Comment:
There is an open PR https://github.com/grafana/pyroscope/pull/2364/files: we should maybe go that way. What do you think?

Reply:
Yes ok, I'll remove it from this one